### PR TITLE
2671: Change tabroll to use drupal behaviors

### PR DESCRIPTION
**Pull request by cableman**
See https://platform.dandigbib.org/issues/2671

### CODE REVIEW

**Comment by cableman** [comment]
The scrutinizer is about JS and "strict" mode, but i'am not sure if we do that in other places

**Comment by kasperg** [comment]
@cableman: You should `use strict` [the same way we do in many other JavaScript files](https://github.com/ding2/ding2/blob/8b432a677fe439f4c56db951b231cae911287bbf/modules/p2/ding_list/js/ding_list_tabledrag.js#L9).

**Comment by cableman** [comment]
@kasperg yes, I thought so, just wanted your take on it.

File: modules/ding_tabroll/js/ding_tabroll.js [hunk 1]
```diff
@@ -4,50 +4,53 @@
...
+      var tab_selected = false;
+
+      // Check if the tabs lib is loaded before trying to call it.
+      if ($.fn.tabs) {
+        tabroll.tabs({
+          select: function(event, ui) {
+            // Update the mobile navigation drop down.
+            tabroll_select.prop('selectedIndex', ui.index);
+          }
+        }).tabs("rotate", 5000);
```
Comment (kasperg) [inline modules/ding_tabroll/js/ding_tabroll.js]: Nitpicking but this also uses double quotes?

Comment (cableman) [reply modules/ding_tabroll/js/ding_tabroll.js]: Fixed it :-)

File: modules/ding_tabroll/js/jquery-ui-tabs-rotate.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 ;(function($){
+	'use strict';
```
Comment (kasperg) [inline modules/ding_tabroll/js/jquery-ui-tabs-rotate.js]: [This file is third party code](https://github.com/cmcculloh/jQuery-UI-Tabs-Rotate) and should not be modified. If anything it should be removed from the repository, downloaded using Drush Make and loaded through libraries API.

Comment (cableman) [reply modules/ding_tabroll/js/jquery-ui-tabs-rotate.js]: Yes, I see. Reverted the library. It's outside the scope to move it to libraries right now. :-)

**Pull request merged**